OpenOversight/app/templates/partials/officer_add_photos.html

```diff
@@ -0,0 +1,3 @@
+{% if current_user.is_administrator or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
```

Comment:
Is `is_admin_or_coordinator` available for this partial?

Reply:
Yes!